good-citizen.md

```diff
@@ -0,0 +1,50 @@
+# Being a Good Citizen of Exercism 
+
+We want Exercism to be a great place to spend time, whoever you are.
+To ensure that Exercism is that, we place a strong focus on being good community citizens.
+This is formalised in our [Code of Conduct](https://exercism.io/code-of-conduct) but can be summed up simply in one sentence: **Be kind.**
+
+## The words we use
+
+There are lots of ways to embrace being kind within Exercism but the most powerful is to consider how your words affect others.
+
+Words in themselves are meaningless. 
+They are vibrations caused by puffs of air coming out of our lungs.
+However, because of **context**, words are powerful.
+
+It's easy to know how you intend your words to come across, but it's difficult to know how others hear your words.
+What we say is strongly tempered by our experiences in life, our cultural norms, and our primary spoken language.
+At Exercism, we believe it's better to choose words that are **clear** and **unambiguous** across cultures and, where possible, across spoken languages.
+This means being careful when using metaphors, acronyms or phrases that might only be understandable to someone from a specific background or with specific knowledge.
+It also means being open to learning about how people from different cultures might misunderstand your intentions, and changing the words you use accordingly.
+
+One example is that within Exercism we ask people not to use the word "guys" unless explicitely addressing a group of men. 
+While in many places the word "guys" can refer to a group of people irrespective of gender, in many other places it explicitly means "men".
+So, regardless of the intention of the speaker, a person from that cultural background hearing the word "guys" understands the speaker to be only addressing men and, if they are not male, feels excluded.
```

Comment:
```suggestion
So, regardless of the intention of the speaker, a person from that cultural background hearing the word "guys" understands the speaker to be only addressing men and, if they are not male, may feel excluded.
```

Reply:
I'm going to leave this one because if someone is non-male and hearing someone address only males, I think they will always feel excluded. (If they know the person intends to speak in a genderless way, then they are probably choosing to be offended/excluded, in which case I'd make the change to the sentence)